tests/test_pqserver.py

```diff
@@ -142,24 +124,20 @@ def app(db, scope="module"):
         pqserver.engine.dispose()
 
 
-@pytest.fixture
-def client(app, scope="module"):
-    # NOTE: all tests share the same client, app, and database.
-    return TestClient(app)
-
-
-def test_root(client):
-    response = client.get("/")
+def test_root(lsstcomcamsim):
+    response = lsstcomcamsim.get("/")
     result = response.json()
     assert "instruments" in result
     assert "obs_types" in result
     assert "dtypes" in result
 
 
-def test_root2(client):
-    response = client.get("/consdb")
+@pytest.mark.parametrize("lsstcomcamsim", ["cdb_latiss"], indirect=True)
+def test_root2(lsstcomcamsim, request):
```

Comment:
It's not really root anymore if it's /consdb/<parameterized>, right? 
Can you name it something more descriptive?

Reply:
root2 => approot?